library/conn_bridge.c

```diff
@@ -214,8 +307,36 @@ static void on_ziti_write(ziti_connection conn, ssize_t status, void *ctx) {
     }
 }
 
+void on_udp_input(uv_udp_t *udp, ssize_t len, const uv_buf_t *b, const struct sockaddr *addr, unsigned int flags) {
+    struct ziti_bridge_s *br = udp->data;
+
+    if (br->idle_timer) { // reset idle timer
+        uv_timer_start(br->idle_timer, on_bridge_idle, br->idle_timeout, 0);
+    }
+
+    if (len > 0) {
+        ziti_write(br->conn, b->base, len, on_ziti_write, b->base);
+    } else {
+        pool_return_obj(b->base);
+        if (len == UV_ENOBUFS) {
+            if (!br->input_throttle) {
+                ZITI_LOG(TRACE, "stalled ziti_conn[%d.%d]", br->conn->ziti_ctx->id, br->conn->conn_id);
```

Comment:
something that "stalled" seems like it should be a warn? trace seems really low?

Reply:
it is not an error condition -- it just means that uplink(to ER) is slower than local connection -- which is true most of the time